### PR TITLE
manually 404 on tableau

### PR DESCRIPTION
**Pull request by sravfeyn**

### CODE REVIEW

File: custom/icds_reports/views.py [hunk 1]
```diff
@@ -116,6 +117,9 @@ def couch_user(self):
         return self.request.couch_user
 
     def get_context_data(self, **kwargs):
+        if settings.SERVER_ENVIRONMENT == 'icds-new':
```
Comment (dannyroberts) [inline custom/icds_reports/views.py]: Can you use the new ICDS_ENVS setting variable as introduced here: https://github.com/dimagi/commcare-hq/pull/18934/files?

File: custom/icds_reports/reports/reports.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from __future__ import absolute_import
 from django.urls import reverse
-
+from django.http import Http404
```
[the remark this answers is not in the document]
Comment (dannyroberts) [reply custom/icds_reports/reports/reports.py]: I think this is real/easily addressable.

**Review by dannyroberts — CHANGES_REQUESTED** [review]

File: custom/icds_reports/views.py (hunk 1 above)
Comment (sravfeyn) [reply custom/icds_reports/views.py]: I thought it would be nice to keep the old one ON, in case GS wants to have a look at old one for any reason.

Comment (dannyroberts) [reply custom/icds_reports/views.py]: oh... but isn't the old one just totally down now? But okay I suppose that's fine.

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**